setup.py

```diff
@@ -30,59 +49,21 @@
     "test": [
         "factory-boy==2.11.1",
         "hypothesis>=5,<6",
```

Comment:
Do we want to drop upper pins in this section too for consistency?

Reply:
All upper pins gone except for `towncrier` and `hypothesis`.